lab_1_classify_by_unigrams/main.py

```diff
@@ -80,6 +158,23 @@ def preprocess_profile(profile: dict) -> dict[str, str | dict] | None:
     :return: a dict with a lower-cased loaded profile
     with relative frequencies without unnecessary ngrams
     """
+    if not (
+            isinstance(profile, dict) and 'name' in profile
+            and 'freq' in profile and 'n_words' in profile
+    ):
+        return None
+
+    preprocessed_profile = {'name': profile['name'], 'freq': {}}
+    unigrams = profile['n_words'][0]
+    for token in profile['freq']:
+        if token.lower() in preprocessed_profile['freq']:
+            preprocessed_profile['freq'][token.lower()] += \
+                profile['freq'][token] / profile['n_words'][0]
```

Comment:
It should be more readable if you save `profile['freq'][token]`, `token.lower()` to a variables

Reply:
you create `unigrams` and do not use it here. Why?